plugins/parsers/csv/parser_test.go

```diff
@@ -849,14 +849,14 @@ func TestParseMetadataSeparators(t *testing.T) {
 	p := &Parser{
 		ColumnNames:        []string{"a", "b"},
 		MetadataRows:       0,
-		MetadataSeparators: []string{},
+		MetadataSeparators: make([]string, 0),
```

Comment:
I think this can also be removed as well as the instance below?!

Reply:
Removed.